resources/asciidoctor/lib/copy_images/extension.rb

```diff
@@ -48,12 +49,17 @@ def process_block_image(block)
     end
 
     def process_inline_image(block)
-      return unless block.content_model == :simple
+      process_inline_image_from_source block, block.source if block.content_model == :simple
+      process_inline_image_from_converted block, block.text if
+        block.context == :list_item && block.parent.context == :olist
+    end
```

Comment:
You might want to reverse the syntax and write the `if` statement first.
I typically only use the post-fix `if` statement if it fits on one line and is fairly simple logic.

Reply:
Got it!